README.md

```diff
@@ -5,6 +5,30 @@
 This is a Rails application to enable citizens to check when their convictions are spent.
 It is based on software patterns developed for the [C100 Application][c100-application].
 
+### Documentation for calculations of convictions
+
+This service follows several rules to calculate spent dates of multiple convictions,
+we have documented all of the scenarios known to us in our tests:
+
+- [spec/services/calculators/multiples/given_scenarios_spec.rb](spec/services/calculators/multiples/given_scenarios_spec.rb)
```

Comment:
I'm not sure if this needs a full link to make it work on GitHub.

Reply:
I think it works. You can see by going here:
https://github.com/ministryofjustice/disclosure-checker/blob/a64194c666c858dd8f483157a9d51f29a6e22681/README.md

There is an extra bullet but other than that looks perfect and links work!

Reply:
Ah! Did not save the file when I added the last link (to the spreadsheet), fixed now